lstchain/reco/tests/test_r0_to_dl1.py

```diff
@@ -16,13 +20,11 @@ def test_rescale_dl1_charge():
     for tel_id in tel_ids:
         np.testing.assert_allclose(event.dl1.tel[tel_id].image, images[tel_id]*rescaling_factor)
 
+
 def test_r0_to_dl1_nsb_tuning(tmp_path, mc_gamma_testfile):
-    from lstchain.reco.r0_to_dl1 import r0_to_dl1
-    from lstchain.io import standard_config
-    import os
-    config = standard_config
+    config = deepcopy(standard_config)
```

Comment:
why is this `deepcopy` needed?

Reply:
The tests mutates the config, so to not change this standard_config object for every following test, the deepcopy is there